### ISSUE TITLE
The two papers have inconsistencies in their explanations of DC

### ISSUE THREAD

**Issue by btt-creator** [Tencent/TFace#119]
Hi，author！thanks your contribution on the PPFP! when i was reading “Privacy-Preserving Face Recognition with Learnable Privacy Budgets in Frequency Domain”， it mentioned that when we remove the DC component, it is impossible to reconstruct the image without knowing the DC component.But in this paper "Privacy-Preserving Face Recognition Using Random Frequency Components",the article states that it is possible to reconstruct the original image using only high-frequency information.Could you please explain how to reconstruct using only high-frequency information?

**Comment by yxmikemi** [comment]
Hi! For the attacker, successful reconstruction depends on various factors, such as training resources (e.g., the amount of training data) and strategy (e.g., loss, optimizer, learning rate). In PartialFace (the "random component" paper), we examine a more advanced attacker that uses a smaller learning rate and leverages a larger training dataset compared to DCTDP (the "learnable budget" paper) and DuetFace (another paper). Empirically, this leads to a more successful recovery.
